### PR TITLE
Feat/home page

### LINKED ISSUE #95: Home page 1.1 (logged out)
image

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**
Closes #95 

Note: Only merge after PR #96

### CODE REVIEW

**Comment by adamjonas** [comment]
Nit: would like to be able to close the accordion with a click on the down arrow.

**Comment by adamjonas** [comment]
I'm good to deploy once someone has looked over the code.

I'd like this page to be the landing page (`/`) for logged-out users rather than `/home` and also serve it at `/tutorial` which should be an option on the pulldown menu for logged-in users.

This is a big upgrade for the product so let's get it 🚢 ed and then add the bells and whistles based on user feedback. Good job.

File: src/components/home/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              alignSelf="center"
+              variant="outline"
+              colorScheme="blue"
+              onClick={getStarted}
+            >
+              Get Started
+            </Button>
+          </Flex>
+          <Box flex="1 1 50%" id="hero-youtube-player">
+            {/* <div className="iframe-wrapper">
```
Comment (Extheoisah) [inline src/components/home/Hero.tsx:49]: remove unused code?

File: src/components/home/YoutubeComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    // prevent flickering
+    setTimeout(() => {
+      setPlayer(e.target);
+    }, 500);
+  };
+
+  return (
+    <div style={Boolean(player) ? { opacity: 1 } : { opacity: 0 }}>
+      <YouTube
+        videoId="YNIFm0QFAuA"
```
Comment (Extheoisah) [inline src/components/home/YoutubeComponent.tsx:27]: is this a constant id that does not change? If so, it shouldn't be be hardcoded but in a constants file

File: src/components/home/YoutubePortal.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  useEffect(() => {
+    heroYoutubeEmbed.current = document.getElementById("hero-youtube-player");
+  }, []);
+
+  if (!heroYoutubeEmbed.current) return null;
+
+  return (
+    <>
+      {createPortal(children, heroYoutubeEmbed.current)}
+      {/* {
```
Comment (Extheoisah) [inline src/components/home/YoutubePortal.tsx:17]: remove unused code?

File: src/components/home/YoutubePortal.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useRef } from "react";
+import { createPortal } from "react-dom";
+
+const wrapperId = "home-media-viewer";
+
+const YoutubePortal = ({ children }: { children: React.ReactNode }) => {
+  const heroYoutubeEmbed = useRef<HTMLElement | null>(null);
```
Comment (Extheoisah) [inline src/components/home/YoutubePortal.tsx:7]: why do we need this ref?

File: src/components/tables/TableItems.tsx [hunk 5]
```diff
@@ -263,3 +269,67 @@ export const ArchiveButton = ({
...
+      {/* <GroupedLinks data={data} /> */}
+    </>
+  );
+};
+
+export const GroupedLinks = ({ data }: { data: ReviewTranscript }) => {
+  const { pr_url } = data.review!;
+  let publishUrl = "";
+  const isPublished = data.review?.mergedAt;
+  // const isPublished =
```
Comment (Extheoisah) [inline src/components/tables/TableItems.tsx:297]: remove duplicate variables and commented code

File: src/styles/globals.css [hunk 6]
```diff
@@ -33,3 +33,45 @@ a {
...
+
+.iframe-wrapper iframe {
+  position: absolute;
+  width: 100% !important;
+  height: 100%;
+  border-radius: 0.5em;
+  box-shadow: 10px 10px 40px 5px rgba(0, 0, 0, 0.2);
+}
+
+/* #home-media-viewer {
```
Comment (Extheoisah) [inline src/styles/globals.css:52]: remove unused css?

**Review by Extheoisah — CHANGES_REQUESTED** [review]
pr looks good. Just a few comments.

**Comment by Emmanuel-Develops** [comment]
@Extheoisah all the unused code are meant for a feature, but because of time restraint we are pushing what we have now. Then adding the feature in the next PR.

For the accordion I'll have to debug that. What browser are you using?

**Comment by Extheoisah** [comment]
> @Extheoisah all the unused code are meant for a feature, but because of time restraint we are pushing what we have now. Then adding the feature in the next PR.
> 
> For the accordion I'll have to debug that. What browser are you using?

Safari. Let me try on chrome.

File: src/components/home/YoutubePortal.tsx (hunk 4 above)
Comment (Emmanuel-Develops) [reply src/components/home/YoutubePortal.tsx:7]: It portals into the heroSection

**Comment by Extheoisah** [comment]
> > @Extheoisah all the unused code are meant for a feature, but because of time restraint we are pushing what we have now. Then adding the feature in the next PR.
> > For the accordion I'll have to debug that. What browser are you using?
> 
> Safari. Let me try on chrome.

Okay, it works seamlessly on chrome. The problem with Safari can be fixed later. Should be a loading issue.

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**